File: FITCCRS2App.Models/Models/Agenda.cs
namespace FITCCRS2App.Models.Models
{
	public class Agenda
	{
        public int AgendaId { get; set; }
        public int? Dan { get; set; }
        public int? TakmicenjeId { get; set; }

        public virtual Takmicenje Takmicenje { get; set; }= new ();
    }
}

File: FITCC.UI/mobile_fitcc/lib/Models/agenda.dart
import 'package:json_annotation/json_annotation.dart';

part 'agenda.g.dart';

@JsonSerializable(explicitToJson: true)
class Agenda {
  int agendaId;
  int? dan;
  int? takmicenjeId;
  dynamic takmicenje; 

  Agenda(
     this.agendaId,
    this.dan,
   this.takmicenje, 
   this.takmicenjeId, 
  );

  factory Agenda.fromJson(Map<String, dynamic> json) => _$AgendaFromJson(json);
  Map<String, dynamic> toJson() => _$AgendaToJson(this);
}
